### PR TITLE
cp-246: changed icon size in chat and profile

### PR DESCRIPTION
**Pull request by juanc-escobar**
*made the chat icon a bit smaller and my profile icon a bit larger
 *added functionality to customize each icon if needed. 
 
 
image

### CODE REVIEW

File: frontend/src/libs/components/navigation-menu/libs/constants.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const NORMAL_ICON_WIDTH = 24;
+const NORMAL_ICON_HEIGHT = 24;
+
```
Comment (danilittaa) [inline frontend/src/libs/components/navigation-menu/libs/constants.ts]: I think it would be better to rename these constants to 'DEFAULT_ICON_WIDTH'

**Review by OlenaYatseniuk — DISMISSED** [review]

**Review by BrianSammit — DISMISSED** [review]

File: frontend/src/libs/components/navigation-menu/libs/constants.ts
Comment (v0ldemar01) [inline frontend/src/libs/components/navigation-menu/libs/constants.ts:1]: libs/constants/constants

**Review by v0ldemar01 — CHANGES_REQUESTED** [review]

File: frontend/src/libs/components/navigation-menu-wrapper/libs/constants.ts [hunk 2]
```diff
@@ -12,21 +12,29 @@ const SIDEBAR_ROUTES: Route[] = [
     path: getUrlWithQueryString(AppRoute.MEDITATION, QUERY_STRING_PARAMETERS),
     name: 'meditation',
     icon: 'meditation',
+    iconWidth: 22,
+    iconHeight: 20,
```
Comment (liza-veis) [inline frontend/src/libs/components/navigation-menu-wrapper/libs/constants.ts]: all of them should have the same width and height - 24
```suggestion
```

File: frontend/src/libs/components/navigation-menu/navigation-menu.tsx [hunk 3]
```diff
@@ -38,8 +39,8 @@ const NavigationMenu: React.FC<Properties> = ({ routes }) => {
                     <Icon
                       name={route.icon}
                       color={IconColor.BLUE}
-                      width={24}
-                      height={24}
+                      width={route.iconWidth ?? DEFAULT_ICON_WIDTH}
```
Comment (liza-veis) [inline frontend/src/libs/components/navigation-menu/navigation-menu.tsx]: check that this won't broken icons in other places

**Review by liza-veis — CHANGES_REQUESTED** [review]

**Comment by juanc-escobar** [comment]
adjusted the icons to be all close to 24 x 24 according to what was requested:

image


image

File: frontend/src/libs/types/route/route.type.ts [hunk 4]
```diff
@@ -7,6 +7,8 @@ type Route = {
   path: ValueOf<typeof AppRoute>;
   name: string;
   icon: IconName;
+  iconWidth?: number;
+  iconHeight?: number;
```
Comment (liza-veis) [inline frontend/src/libs/types/route/route.type.ts]:
```suggestion
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

**Review by OlesiaTr — APPROVED** [review]

**Review by dmytro-kucherenko — APPROVED** [review]

**Review by liza-veis — APPROVED** [review]

**Pull request merged**